yarn.lock

```diff
@@ -9940,6 +9998,14 @@ redux@^3.6.0:
     loose-envify "^1.1.0"
     symbol-observable "^1.0.3"
 
+redux@^4.0.4:
```

Comment:
If we go with this implementation I will update our Redux to be in line with this so that we don't need multiple versions.